microsoft-edge/toc.yml

```diff
@@ -361,6 +361,8 @@
               href: webview2/gettingstarted/winui.md
         - name: Concepts
           items:
+            - name: WebView2 developer guide
```

Comment:
> WebView2 developer guide

Can we use "WebView2 Developer Best Practices" in ToC? Developer guide can mean a lot of things and essentially everything we write is developer guide.

Reply:
@liminzhu 
`Best Practices` is an overloaded phrase and should be avoided.

Reply:
Anything you'd recommend besides developer guide? Developer guide feels too generic and doesn't really convey the sense this is a set of best practices we want developers to follow.

Reply:
@liminzhu 
My understanding is this article is meant to convey activities that practices that will aid success, not guarantee success.  The term `best` connotes that it achieves the achieves peak output, instead of striving for a state of `better` than the alternative.  
That being said, the term `guide` or `guidance` is very appropriate.  Phrases like `Improve your WebView2 development` or `Guidance for WebView2 success` or or `Practices for WebView2 success` or `WebView2 developer guide` fit the content.

Reply:
Is there a better alternative than developer guide? I mean...everything we publish is a developer guide, so it's not really saying much about what the doc is about. If someone search "WebView2 developer guide", I'd also want our landing page to show up first as opposed to this.

If there is strong opinion around using the phrase best practice, can we also be consistent across the doc? Right now the title of the doc is "WebView2 development best practices".

Reply:
Maybe "Recommended practices"?